wire-gradle-plugin/README.md

```diff
@@ -0,0 +1,23 @@
+Wire Gradle Plugin
```

Comment:
Sorry for the horrible misunderstanding, but could you bring this in as a separate PR? It’s really awkward to mix so many commits